lib/Myriad/Transport/Redis.pm

```diff
@@ -75,6 +75,9 @@ method configure (%args) {
     $max_pool_count = exists $args{max_pool_count} ? delete $args{max_pool_count} : 10;
     $prefix //= exists $args{prefix} ? delete $args{prefix} : 'myriad';
     $clientside_cache_size = delete $args{client_side_cache_size} if exists $args{client_side_cache_size};
+    $wait_time = exists $args{wait_time} ? delete $args{wait_time} : 15_000;
+    # limit minimum wait time to 100ms
+    $wait_time = 100 if $wait_time < 100;
```

Comment:
added it as a separate line here to make it more readable.

I thought of limiting that from the config itself. However I think its better not to concern config class itself with such things.